apps/marketplace/common-util/Details/index.tsx

```diff
@@ -56,8 +47,8 @@ const getColumns = ({
             onClick={() => openActivityModal(record)}
             style={{ display: 'flex', gap: 8 }}
           >
-            {<AddressLink suffixCount={8} textMinWidth={160} text={text} cannotClick />}
-            <CopyBtn text={text} />
+            <AddressLink suffixCount={8} textMinWidth={160} text={text} cannotClick />
+            <Copy text={text} />
```

Comment:
wonder, why this wasn't initially done via "canCopy" prop in AddressLink

Reply:
the styles weren't matching with figma, didn't wanted to override